swagger.yaml

```diff
@@ -155,5 +317,116 @@ components:
+
+    Leads:
+      type: object
+      description: Lead data from Pipedrive API
+      properties:
+        id:
+          type: string
+        title:
+          type: string
+        interested:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        usedTech:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        add_time:
+          type: string
+          format: date-time
+        update_time:
+          type: string
+          format: date-time
+        next_activity_date:
+          type: string
+          format: date-time
+        label_ids:
+          type: array
+          items:
+            type: string
+    Deals:
+      type: object
+      description: Deal data from Pipedrive API
+      properties:
+        id:
+          type: string
+        title:
+          type: string
+        interested:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        usedTech:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        value:
+          type: number
+        currency:
+          type: string
+        add_time:
+          type: string
+          format: date-time
+        update_time:
+          type: string
+          format: date-time
+        next_activity_date:
+          type: string
+          format: date-time
+        status:
+          type: string
+        next_activity_subject:
+          type: string
+        next_activity_note:
+          type: string
+
+    LeadOrDeal:
+      type: object
+      description: Used when user clicks to see more data from lead or deal. Lead or Deal data from Pipedrive API.
+      properties:
+        id:
+          type: string
+        title:
+          type: string
+        interested:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        usedTech:
+          type: string
+          description: "Might need to be changed to Pipedrive API item-key"
+        add_time:
+          type: string
+          format: date-time
+        update_time:
+          type: string
+          format: date-time
+        next_activity_date:
+          type: string
+          format: date-time
+        status:
+          type: string
+          description: "Added from Deal"
+        next_activity_subject:
+          type: string
+        next_activity_note:
+          type: string
+        value:
+          type: number
+          description: "Added from Deal"
+        currency:
+          type: string
+          description: "Added from Deal"
+        label_ids:
+          type: array
+          items:
+            type: string
+          description: "Added from Lead"
+
+    Interests:
+      type: object
+      description: Adding or removing interest to Pipedrive API
+      properties:
+        pipedrive-interest-key-here:
```

Comment:
Can you clarify this?

Reply:
If you mean the "pipedrive-interest-key-here". The variables in PUT and PATCH send the variable name instead the value and gave me paranoia. But I think only place where the Pipedrives field-key matters is in the PUT/PATCH function.